pkg/eval/mods/file/file_test.go

```diff
@@ -30,7 +30,21 @@ func TestFile(t *testing.T) {
 				What:     "arguments here",
 				ValidLow: 1, ValidHigh: 1, Actual: 0}),
 
-		That(`p = (file:pipe); echo haha > $p; pwclose $p; slurp < $p; prclose $p`).Puts("haha\n"),
+		That("file:prclose").Throws(
+			errs.ArityMismatch{
+				What:     "arguments here",
+				ValidLow: 1, ValidHigh: 1, Actual: 0}),
+
+		That("file:pwclose").Throws(
+			errs.ArityMismatch{
+				What:     "arguments here",
+				ValidLow: 1, ValidHigh: 1, Actual: 0}),
+
+		That(`p = (file:pipe); echo haha > $p; pwclose $p
```

Comment:
Don't wrap strings by embedding a newline. You can also avoid the need for semicolons by splitting the command line into separate statements:

```
That(`p = (file:pipe)`, `echo haha > $p`, `pwclose $p`, `slurp < $p`, `prclose $p`).Puts("haha\n")
```

You can then split the long line at whichever `, ` boundaries make the most sense.